CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Proxy] Fix proxy deploy timeout in case of not correctly get the proxy VM IP [#4927](https://github.com/vatesfr/xen-orchestra/issues/4927) (PR [#4928](https://github.com/vatesfr/xen-orchestra/pull/4928))
```

Comment:
Does it need a changelog entry? Was there a specific error that users could have encountered?

Reply:
Yes, the error is "operation timed out" it means that the deploy timeout.

Reply:
Ok so maybe add to the message, that's what the users actually care about. "This specific error won't happen again".